src/openfermion/measurements/_data_containers.py

```diff
@@ -0,0 +1,99 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""
+Classes to store experimental data for postprocessing.
+"""
+
+
+class QPERoundData(object):
+    """ Results from a single round in a QPE experiment.
+    Attributes:
+        nrot (int): the number of rotations performed in this round
+        finrot (float): the final rotation performed on the ancilla qubit
+            in this round.
+        msmt (bool): the measurement observed in this round
+        tmsmt (bool): if the ancilla is not reset between rounds, the
+            'true' measurement (in this case msmt is calculated as the
+            difference between the tmsmt of different rounds).
+    """
+
+    def __init__(self,
+                 num_rotations,
+                 final_rotation,
+                 measurement,
+                 true_measurement=None):
+        """
+        Args:
+            nrot (int): the number of rotations performed in this round
+            finrot (float): the final rotation performed on the ancilla qubit
+                in this round.
+            msmt (bool): the measurement observed in this round
+            tmsmt (bool): if the ancilla is not reset between rounds, the
```

Comment:
These should be the actual parameter names (`num_rotations` instead of `nrot`)

Reply:
Fixed, thanks for spotting.